x-pack/docs/en/rest-api/graph/explore.asciidoc

```diff
@@ -204,6 +208,7 @@ POST clicklogs/_xpack/graph/_explore
 }
 --------------------------------------------------
 // CONSOLE
+// TEST[setup:clicklogs_index]
 <1> Seed the exploration with a query. This example is searching
 clicklogs for people who searched for the term "midi".
 <2> Identify the vertices to include in the graph. This example is looking for
```

Comment:
I see this query refers to a "query.raw" which is a field not likely to be in the clicklogs index mapping as it is currently defined in the build.gradle file